From 994d2a1605ddeacf7d2305ee65ffb7ae357b5efe Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Wed, 28 Sep 2022 22:25:25 -0300
Subject: [PATCH] Re-order services so that it works even without qtaguid

---
 .../android/server/net/NetworkPolicyManagerService.java   | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/services/core/java/com/android/server/net/NetworkPolicyManagerService.java b/services/core/java/com/android/server/net/NetworkPolicyManagerService.java
index 1ea949ede03d..52dc0f63aa77 100644
--- a/services/core/java/com/android/server/net/NetworkPolicyManagerService.java
+++ b/services/core/java/com/android/server/net/NetworkPolicyManagerService.java
@@ -922,6 +922,10 @@ public class NetworkPolicyManagerService extends INetworkPolicyManager.Stub {
         Trace.traceBegin(Trace.TRACE_TAG_NETWORK, "systemReady");
         final int oldPriority = Process.getThreadPriority(Process.myTid());
         try {
+	    mUsageStats = LocalServices.getService(UsageStatsManagerInternal.class);
+            mAppStandby = LocalServices.getService(AppStandbyInternal.class);
+            mActivityManagerInternal = LocalServices.getService(ActivityManagerInternal.class);
+
             // Boost thread's priority during system server init
             Process.setThreadPriority(Process.THREAD_PRIORITY_FOREGROUND);
             if (!isBandwidthControlEnabled()) {
@@ -929,10 +933,6 @@ public class NetworkPolicyManagerService extends INetworkPolicyManager.Stub {
                 return;
             }
 
-            mUsageStats = LocalServices.getService(UsageStatsManagerInternal.class);
-            mAppStandby = LocalServices.getService(AppStandbyInternal.class);
-            mActivityManagerInternal = LocalServices.getService(ActivityManagerInternal.class);
-
             synchronized (mUidRulesFirstLock) {
                 synchronized (mNetworkPoliciesSecondLock) {
                     updatePowerSaveWhitelistUL();
-- 
2.37.3

