From 0d6237bd9931f4c21f3df2d77b73c25f3eec77b6 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Mon, 26 Dec 2022 05:42:06 +0000
Subject: [PATCH] vendor_lineage: Log privapp-permissions whitelist violations
 instead

---
 config/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index fd235d69..d570a735 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -78,9 +78,9 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     vendor/lineage/config/permissions/org.lineageos.android.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/permissions/org.lineageos.android.xml
 
-# Enforce privapp-permissions whitelist
+# Log privapp-permissions whitelist violations
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-    ro.control_privapp_permissions=enforce
+    ro.control_privapp_permissions=log
 
 # Include AOSP audio files
 include vendor/lineage/config/aosp_audio.mk
-- 
2.25.1

