From 5287b85169971c032ff8b3689358546375e0190c Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Mon, 26 Dec 2022 11:54:04 +0000
Subject: [PATCH] remove su from no-su builds

---
 target/product/base_system.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index add010cff..15724e886 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -396,10 +396,10 @@ PRODUCT_PACKAGES_DEBUG := \
     unwind_reg_info \
     unwind_symbols \
 
-ifeq ($(LINEAGE_BUILD),)
-PRODUCT_PACKAGES_DEBUG += \
-    su
-endif
+#ifeq ($(LINEAGE_BUILD),)
+#PRODUCT_PACKAGES_DEBUG += \
+#    su
+#endif
 
 # The set of packages whose code can be loaded by the system server.
 PRODUCT_SYSTEM_SERVER_APPS += \
-- 
2.25.1

