From c0148ccd9f01c20df289ea22557d7d2b8a9ff16a Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Tue, 27 Dec 2022 05:39:44 +0000
Subject: [PATCH] Make broken copy headers the default

---
 core/board_config.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/board_config.mk b/core/board_config.mk
index dc50a689a..a7cb809ff 100644
--- a/core/board_config.mk
+++ b/core/board_config.mk
@@ -207,6 +207,7 @@ endif
 # ###############################################################
 $(foreach v,$(_build_broken_var_list),$(eval $(v) :=))
 BUILD_BROKEN_NINJA_USES_ENV_VARS :=
+BUILD_BROKEN_USES_BUILD_COPY_HEADERS := true
 
 # Boards may be defined under $(SRC_TARGET_DIR)/board/$(TARGET_DEVICE)
 # or under vendor/*/$(TARGET_DEVICE).  Search in both places, but
-- 
2.25.1

