From bf54fb2cf068a02874b7333c0bf21c0f3246d1d9 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Thu, 29 Sep 2022 22:46:01 -0300
Subject: [PATCH] Every voldmanaged storage is adoptable

---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 5b1bdf7..e33570b 100644
--- a/main.cpp
+++ b/main.cpp
@@ -273,7 +273,7 @@ static int process_config(VolumeManager* vm, VoldConfigs* configs) {
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 configs->has_adoptable = true;
             }
-- 
2.37.3

