From 02e2bb259bae24c35880be8ee3324bc04837fabc Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Sat, 12 Nov 2022 07:15:31 +0000
Subject: [PATCH] Improve-SELinux-policy-workaround-on-device-phh-treble

---
 libsepol/cil/src/cil_build_ast.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libsepol/cil/src/cil_build_ast.c b/libsepol/cil/src/cil_build_ast.c
index 023fd6c7..61c8864b 100644
--- a/libsepol/cil/src/cil_build_ast.c
+++ b/libsepol/cil/src/cil_build_ast.c
@@ -141,7 +141,6 @@ int cil_add_decl_to_symtab(struct cil_db *db, symtab_t *symtab, hashtab_key_t ke
 		/* multiple_decls is enabled and works for this datum type, add node */
 		cil_list_append(prev->nodes, CIL_NODE, node);
 		node->data = prev;
-		return SEPOL_EEXIST;
 	}
 
 	return SEPOL_OK;
-- 
2.25.1

