From facd6d2ddfeee6f90005bbe864c06fd37001a5ad Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Tue, 27 Dec 2022 05:32:54 +0000
Subject: [PATCH] Add missing texfat FS driver name

---
 main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main.cpp b/main.cpp
index e33570b..e50872c 100644
--- a/main.cpp
+++ b/main.cpp
@@ -72,6 +72,7 @@ int main(int argc, char** argv) {
     ATRACE_BEGIN("main");
 
     LOG(DEBUG) << "Detected support for:"
+               << (android::vold::IsFilesystemSupported("texfat") ? " texfat" : "")
                << (android::vold::IsFilesystemSupported("exfat") ? " exfat" : "")
                << (android::vold::IsFilesystemSupported("ext4") ? " ext4" : "")
                << (android::vold::IsFilesystemSupported("f2fs") ? " f2fs" : "")
-- 
2.25.1

