From bb2b66de553fef9ba4e073f9a99fdbf309846104 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sat, 1 Oct 2022 02:48:17 +0000
Subject: [PATCH] Don't build init.vndk-nodef.rc

---
 target/product/gsi_release.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/target/product/gsi_release.mk b/target/product/gsi_release.mk
index 74501cd1f..28700cd39 100644
--- a/target/product/gsi_release.mk
+++ b/target/product/gsi_release.mk
@@ -59,8 +59,7 @@ PRODUCT_PACKAGES += com.android.apex.cts.shim.v1_with_prebuilts.flattened
 # GSI specific tasks on boot
 PRODUCT_PACKAGES += \
     gsi_skip_mount.cfg \
-    init.gsi.rc \
-    init.vndk-nodef.rc \
+    init.gsi.rc
 
 # Support additional VNDK snapshots
 PRODUCT_EXTRA_VNDK_VERSIONS := \
-- 
2.34.1

