From 5ec41e7cd3c5672bf873f30f8eac5dc17d48c62d Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Sat, 3 Sep 2022 14:35:07 +0000
Subject: [PATCH 2/3] changes for configuring makefile for crDroid

---
 base.mk       | 38 ++++++++++++++++++++++++++++++++++----
 board-base.mk |  4 +++-
 2 files changed, 37 insertions(+), 5 deletions(-)

diff --git a/base.mk b/base.mk
index af27de5..d00d356 100644
--- a/base.mk
+++ b/base.mk
@@ -8,10 +8,6 @@ PRODUCT_COPY_FILES := \
 	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-
 BOARD_PLAT_PRIVATE_SEPOLICY_DIR += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += device/phh/treble/overlay
 
@@ -50,6 +46,7 @@ PRODUCT_COPY_FILES += \
 #   Provide default libnfc-nci.conf file for devices that does not have one in
 #   vendor/etc
 PRODUCT_COPY_FILES += \
+	system/nfc/conf/libnfc-nci.conf:system/etc/libnfc-nci.conf \
 	device/phh/treble/nfc/libnfc-nci.conf:system/phh/libnfc-nci-oreo.conf \
 	device/phh/treble/nfc/libnfc-nci-huawei.conf:system/phh/libnfc-nci-huawei.conf
 
@@ -233,8 +230,41 @@ PRODUCT_COPY_FILES += \
 	frameworks/av/services/audiopolicy/config/a2dp_audio_policy_configuration_7_0.xml:system/etc/a2dp_audio_policy_configuration_7_0.xml \
 	frameworks/av/services/audiopolicy/config/a2dp_audio_policy_configuration.xml:system/etc/a2dp_audio_policy_configuration.xml \
 
+PRODUCT_PACKAGES += \
+	android.hardware.vibrator@1.3-service.meizu
+
 include build/make/target/product/gsi_release.mk
 
 # Protect deskclock from power save
 PRODUCT_COPY_FILES += \
 	device/phh/treble/files/com.android.deskclock_whitelist.xml:system/etc/sysconfig/com.android.deskclock_whitelist.xml
+
+$(call inherit-product, vendor/lineage/config/common.mk)
+$(call inherit-product, vendor/lineage/config/common_full.mk)
+$(call inherit-product, vendor/lineage/config/BoardConfigSoong.mk)
+$(call inherit-product, vendor/lineage/config/BoardConfigLineage.mk)
+$(call inherit-product, device/lineage/sepolicy/common/sepolicy.mk)
+#$(call inherit-product, vendor/lineage/config/bootanimation.mk)
+-include vendor/lineage/build/core/config.mk
+TARGET_NO_KERNEL_OVERRIDE := true
+TARGET_NO_KERNEL_IMAGE := true
+SELINUX_IGNORE_NEVERALLOWS := true
+TARGET_BOOT_ANIMATION_RES := 1080
+
+TARGET_USES_PREBUILT_VENDOR_SEPOLICY := true
+TARGET_HAS_FUSEBLK_SEPOLICY_ON_VENDOR := true
+
+TARGET_FACE_UNLOCK_SUPPORTED := true
+
+PRODUCT_PACKAGES += \
+    GCamGOPrebuilt
+
+PRODUCT_PACKAGES += \
+    Dialer
+
+PRODUCT_PACKAGES += \
+    libaptX_encoder \
+    libaptXHD_encoder
+
+PRODUCT_PACKAGES += \
+    Gallery
diff --git a/board-base.mk b/board-base.mk
index 1ddacaf..b9ac5aa 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -5,5 +5,7 @@ DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
 
 BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
-
+BUILD_BROKEN_DUP_RULES := true
+#BOARD_EXT4_SHARE_DUP_BLOCKS := true
+TARGET_NO_KERNEL_OVERRIDE := true
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
-- 
2.25.1

