From c055d562f29ec7d6b10d84a6068753e7a9072c65 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Thu, 29 Sep 2022 00:11:56 -0300
Subject: [PATCH] On Samsung, we need to send a hack-message to HAL to get all
 Sensors

---
 services/sensorservice/SensorDevice.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/services/sensorservice/SensorDevice.cpp b/services/sensorservice/SensorDevice.cpp
index de050e02d0..c3c78de6f1 100644
--- a/services/sensorservice/SensorDevice.cpp
+++ b/services/sensorservice/SensorDevice.cpp
@@ -37,6 +37,7 @@
 #include <cinttypes>
 #include <cstddef>
 #include <thread>
+#include <android-base/properties.h>
 
 using namespace android::hardware::sensors;
 using android::hardware::Return;
@@ -77,6 +78,8 @@ SensorDevice::SensorDevice() {
 }
 
 void SensorDevice::initializeSensorList() {
+    if(::android::base::GetBoolProperty("persist.sys.phh.samsung_sensors", false))
+        setMode(5555);
     if (mHalWrapper == nullptr) {
         return;
     }
-- 
2.37.3

