From 63808b82a5673f9cfa9454e4914cd25c2ec9d688 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Wed, 19 Oct 2022 16:37:39 +0000
Subject: [PATCH] Add-broken-duplicate-rules-flag

---
 board-base.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/board-base.mk b/board-base.mk
index 1ddacaf..064e06f 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -6,4 +6,8 @@ DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
 BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 
+BUILD_BROKEN_DUP_RULES := true
+#BOARD_EXT4_SHARE_DUP_BLOCKS := true
+TARGET_NO_KERNEL_OVERRIDE := true
+
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
-- 
2.25.1

