From 13ce02c22b72bc4a8ae77798d2ba42f6fc921efd Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Tue, 12 Jul 2022 01:22:29 +0000
Subject: [PATCH] Adapt Securize to Dynamic Root

---
 phh-securize.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/phh-securize.sh b/phh-securize.sh
index 80b46d7..26b63f7 100644
--- a/phh-securize.sh
+++ b/phh-securize.sh
@@ -24,12 +24,13 @@ done
 mount -o remount,rw $MOUNTPOINT
 remount
 
+sed -i '/phhroot/,$d' $SYSTEM/bin/phh-prop-handler.sh
 touch $SYSTEM/phh/secure
 rm $SYSTEM/xbin/su
 rm $SYSTEM/bin/phh-su
 rm $SYSTEM/etc/init/su.rc
 rm $SYSTEM/bin/phh-securize.sh
-rm -Rf $SYSTEM/{app,priv-app}/me.phh.superuser/
+rm -Rf $SYSTEM/priv-app/SuperUser
 rm -Rf /data/su || true
 mount -o remount,ro $MOUNTPOINT
 sync
-- 
2.17.1

