From 89bae248aa7b6964c13172fd3106f3f447e2af18 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Tue, 12 Jul 2022 01:34:25 +0000
Subject: [PATCH] Adapt Securize due PixelPropUtils

---
 rw-system.sh | 122 ++++++++++++++++++++++++++-------------------------
 1 file changed, 63 insertions(+), 59 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index 3a17730..5fb3677 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -724,66 +724,70 @@ copyprop() {
         resetprop_phh "$1" "$(getprop "$2")"
     fi
 }
-if [ -f /system/phh/secure ] || [ -f /metadata/phh/secure ];then
-    copyprop ro.build.device ro.vendor.build.device
-    copyprop ro.system.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.bootimage.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.build.device ro.vendor.product.device
-    copyprop ro.product.system.device ro.vendor.product.device
-    copyprop ro.product.device ro.vendor.product.device
-    copyprop ro.product.system.device ro.product.vendor.device
-    copyprop ro.product.device ro.product.vendor.device
-    copyprop ro.product.system.name ro.vendor.product.name
-    copyprop ro.product.name ro.vendor.product.name
-    copyprop ro.product.system.name ro.product.vendor.device
-    copyprop ro.product.name ro.product.vendor.device
-    copyprop ro.system.product.brand ro.vendor.product.brand
-    copyprop ro.product.brand ro.vendor.product.brand
-    copyprop ro.product.system.model ro.vendor.product.model
-    copyprop ro.product.model ro.vendor.product.model
-    copyprop ro.product.system.model ro.product.vendor.model
-    copyprop ro.product.model ro.product.vendor.model
-    copyprop ro.build.product ro.vendor.product.model
-    copyprop ro.build.product ro.product.vendor.model
-    copyprop ro.system.product.manufacturer ro.vendor.product.manufacturer
-    copyprop ro.product.manufacturer ro.vendor.product.manufacturer
-    copyprop ro.system.product.manufacturer ro.product.vendor.manufacturer
-    copyprop ro.product.manufacturer ro.product.vendor.manufacturer
-    (getprop ro.vendor.build.security_patch; getprop ro.keymaster.xxx.security_patch) |sort |tail -n 1 |while read v;do
-        [ -n "$v" ] && resetprop_phh ro.build.version.security_patch "$v"
-    done
-
-    resetprop_phh ro.build.tags release-keys
-    resetprop_phh ro.boot.vbmeta.device_state locked
-    resetprop_phh ro.boot.verifiedbootstate green
-    resetprop_phh ro.boot.flash.locked 1
-    resetprop_phh ro.boot.veritymode enforcing
-    resetprop_phh ro.boot.warranty_bit 0
-    resetprop_phh ro.warranty_bit 0
-    resetprop_phh ro.debuggable 0
-    resetprop_phh ro.secure 1
-    resetprop_phh ro.build.type user
-    resetprop_phh ro.build.selinux 0
-
-    resetprop_phh ro.adb.secure 1
-    setprop ctl.restart adbd
-
-    # Hide system/xbin/su
-    mount /mnt/phh/empty_dir /system/xbin
-    mount /mnt/phh/empty_dir /system/app/me.phh.superuser
-    mount /system/phh/empty /system/xbin/phh-su
-else
-    mkdir /mnt/phh/xbin
-    chmod 0755 /mnt/phh/xbin
-    chcon u:object_r:system_file:s0 /mnt/phh/xbin
-
-    #phh-su will bind over this empty file to make a real su
-    touch /mnt/phh/xbin/su
-    chcon u:object_r:system_file:s0 /mnt/phh/xbin/su
 
-    mount -o bind /mnt/phh/xbin /system/xbin
-fi
+# API 27
+copyprop ro.product.brand ro.vendor.product.brand
+copyprop ro.product.system.brand ro.vendor.product.brand
+copyprop ro.product.product.brand ro.vendor.product.brand
+copyprop ro.product.system_ext.brand ro.vendor.product.brand
+copyprop ro.product.product.device ro.vendor.product.device
+copyprop ro.product.system.device ro.vendor.product.device
+copyprop ro.product.system_ext.device ro.vendor.product.device
+copyprop ro.product.device ro.vendor.product.device
+copyprop ro.product.manufacturer ro.vendor.product.manufacturer
+copyprop ro.product.system.manufacturer ro.vendor.product.manufacturer
+copyprop ro.product.product.manufacturer ro.vendor.product.manufacturer
+copyprop ro.product.system_ext.manufacturer ro.vendor.product.manufacturer
+copyprop ro.build.product ro.vendor.product.model
+copyprop ro.product.model ro.vendor.product.model
+copyprop ro.product.system.model ro.vendor.product.model
+copyprop ro.product.product.model ro.vendor.product.model
+copyprop ro.product.system_ext.model ro.vendor.product.model
+copyprop ro.product.name ro.vendor.product.name
+copyprop ro.product.system.name ro.vendor.product.name
+copyprop ro.product.product.name ro.vendor.product.name
+copyprop ro.product.system_ext.name ro.vendor.product.name
+
+# API28+
+copyprop ro.product.brand ro.product.vendor.brand
+copyprop ro.product.system.brand ro.product.vendor.brand
+copyprop ro.product.product.brand ro.product.vendor.brand
+copyprop ro.product.system_ext.brand ro.product.vendor.brand
+copyprop ro.product.device ro.product.vendor.device
+copyprop ro.product.system.device ro.product.vendor.device
+copyprop ro.product.product.device ro.product.vendor.device
+copyprop ro.product.system_ext.device ro.product.vendor.device
+copyprop ro.product.manufacturer ro.product.vendor.manufacturer
+copyprop ro.product.system.manufacturer ro.product.vendor.manufacturer
+copyprop ro.product.product.manufacturer ro.product.vendor.manufacturer
+copyprop ro.product.system_ext.manufacturer ro.product.vendor.manufacturer
+copyprop ro.build.product ro.product.vendor.model
+copyprop ro.product.model ro.product.vendor.model
+copyprop ro.product.system.model ro.product.vendor.model
+copyprop ro.product.product.model ro.product.vendor.model
+copyprop ro.product.system_ext.model ro.product.vendor.model
+copyprop ro.product.name ro.product.vendor.name
+copyprop ro.product.system.name ro.product.vendor.name
+copyprop ro.product.product.name ro.product.vendor.name
+copyprop ro.product.system_ext.name ro.product.vendor.name
+
+# FINGERPRINTS
+copyprop ro.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.bootimage.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.system.build.fingerprint ro.vendor.build.fingerprint   
+copyprop ro.product.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.system_ext.build.fingerprint ro.vendor.build.fingerprint
+
+# COMMON
+resetprop_phh ro.build.tags release-keys
+resetprop_phh ro.boot.vbmeta.device_state locked
+resetprop_phh ro.boot.verifiedbootstate green
+resetprop_phh ro.boot.flash.locked 1
+resetprop_phh ro.boot.veritymode enforcing
+resetprop_phh ro.boot.warranty_bit 0
+resetprop_phh ro.warranty_bit 0
+resetprop_phh ro.build.type user
+resetprop_phh ro.build.selinux 0
 
 for abi in "" 64;do
     f=/vendor/lib$abi/libstagefright_foundation.so
-- 
2.17.1

